.github/workflows/continous-integration.yml

```diff
@@ -25,6 +25,7 @@ jobs:
     - uses: actions/setup-node@v2
       with:
         node-version: 14
+        cache: 'npm'
```

Comment:
According to https://github.com/actions/setup-node, we can set the "cache" flag, which uses https://github.com/actions/toolkit/tree/main/packages/cache under the hood!